### DISCUSSION TITLE
Does -a support for zing jvm?

### DISCUSSION THREAD

**Discussion by zct** [async-profiler/async-profiler#1026] [Q&A] [answered]
The jvm what i used is zing jvm
> -a annotate JIT compiled methods with _[j], inlined methods with _[i], interpreted methods with _[0] and C1 compiled methods with _[1]

 I try to add -a parameter in the profiler cmd, but the result flamegraph don't show any differences with before, don't have any information with jit methods and inline methods

**Comment by apangin** [comment, marked as answer]
`-a` is relevant to collapsed output format only, it has no effect on a flame graph.
Flame graph displays compiled/interpreted/inlined methods by default, but this is not supported on Zing JVM, since Zing does not export required info VM structures info.

**Reply by zct** [reply]
thanks for your reply
